### DISCUSSION TITLE
登録サンキューメールのURL

### DISCUSSION THREAD

**Discussion by netmarkjp** [topotal/waroom-roadmap#13] [Others]
Bugじゃないけど改善要望...

Invitation Link	として表記されているのは https://api.beta.waroom.com/〜 だけど、実際の遷移先は https://u24647278.ct.sendgrid.net/〜 だったので、おっ、ってなりました。

![image](https://user-images.githubusercontent.com/87943/150046835-cd974cde-df3c-4a47-803b-962326b1586d.png)

**Comment by sawa-zen** [comment]
ありがとうございます。 ~~Sendgrid で設定したメールのテンプレートを確認してみます 🙏🏻~~

**Reply by sawa-zen** [reply]
Sendgrid で送信したメールのリンクのクリックをトラッキングしていたんですが、その際に書き換われるドメインを sendgrid.net から waroom.com のものになるように修正しました 🙏🏻  
次回以降送信されるメールではこのアラートがでなくなったと思います！
報告ありがとうございました 😄
